package.json

```diff
@@ -109,5 +108,6 @@
   "resolutions": {
     "@types/react": "18.2.37",
     "typescript": "4.7.4"
-  }
+  },
+  "packageManager": "yarn@3.7.0"
```

Comment:
This is an unofficial property I guess…?

https://docs.npmjs.com/cli/v10/configuring-npm/package-json